### PR TITLE
🐛 Source Google Ads: add 'asset' Resource to full refresh custom tables

### LINKED ISSUE #25620: Source Google Ads: Connector does not support the "Asset" Resource in "Custom GAQL Queries" input
### Connector Name

airbyte/source-google-ads

### Connector Version

0.2.14

### What step the error happened?

Configuring a new connector

### Revelant information

The "Asset" Resource from Google Ads does not support the field "segments.date" because it only works in full_refresh mode.
But following:
- this PR #22703
- its linked issue #14446
- and this specific [comment](https://github.com/airbytehq/airbyte/issues/14446#issuecomment-1442230542) in the issue

the connector only allows the Resources "geo_target_constant" and "custom_audience". 
(See here: https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-google-ads/source_google_ads/source.py#L40)

image

Until a better and generic solution had been found (maybe something like #25284) in order to handle all the Resources (incremental and full_refresh types) from Google Ads, I will just add the "asset" Resource to the FULL_REFRESH_CUSTOM_TABLES.
Other tables can be easily added to the FULL_REFRESH_CUSTOM_TABLES in the future.

### Relevant log output

```shell
2023-04-27 13:39:25 [1;31mERROR[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):144 - Traceback (most recent call last):
  File "/airbyte/integration_code/source_google_ads/source.py", line 113, in check_connection
    response = google_api.send_request(str(query), customer_id=customer.id)
  File "/usr/local/lib/python3.9/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
  File "/airbyte/integration_code/source_google_ads/google_ads.py", line 64, in send_request
    return [self.ga_service.search(search_request)]
  File "/usr/local/lib/python3.9/site-packages/google/ads/googleads/v13/services/services/google_ads_service/client.py", line 3460, in search
    response = rpc(
  File "/usr/local/lib/python3.9/site-packages/google/api_core/gapic_v1/method.py", line 113, in __call__
    return wrapped_func(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/google/api_core/grpc_helpers.py", line 72, in error_remapped_callable
    return callable_(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/grpc/_interceptor.py", line 247, in __call__
    response, ignored_call = self._with_call(request,
  File "/usr/local/lib/python3.9/site-packages/grpc/_interceptor.py", line 290, in _with_call
    return call.result(), call
  File "/usr/local/lib/python3.9/site-packages/grpc/_interceptor.py", line 146, in result
    raise self._exception
  File "/usr/local/lib/python3.9/site-packages/grpc/_interceptor.py", line 274, in continuation
    response, call = self._thunk(new_method).with_call(
  File "/usr/local/lib/python3.9/site-packages/grpc/_interceptor.py", line 301, in with_call
    return self._with_call(request,
  File "/usr/local/lib/python3.9/site-packages/grpc/_interceptor.py", line 290, in _with_call
    return call.result(), call
  File "/usr/local/lib/python3.9/site-packages/grpc/_interceptor.py", line 146, in result
    raise self._exception
  File "/usr/local/lib/python3.9/site-packages/grpc/_interceptor.py", line 274, in continuation
    response, call = self._thunk(new_method).with_call(
  File "/usr/local/lib/python3.9/site-packages/grpc/_interceptor.py", line 301, in with_call
    return self._with_call(request,
  File "/usr/local/lib/python3.9/site-packages/grpc/_interceptor.py", line 287, in _with_call
    call = self._interceptor.intercept_unary_unary(continuation,
  File "/usr/local/lib/python3.9/site-packages/google/ads/googleads/interceptors/exception_interceptor.py", line 99, in intercept_unary_unary
    self._handle_grpc_failure(response)
  File "/usr/local/lib/python3.9/site-packages/google/ads/googleads/interceptors/exception_interceptor.py", line 71, in _handle_grpc_failure
    raise self._get_error_from_response(response)
google.ads.googleads.errors.GoogleAdsException: (<_InactiveRpcError of RPC that terminated with:
	status = StatusCode.INVALID_ARGUMENT
	details = "Request contains an invalid argument."
	debug_error_string = "UNKNOWN:Error received from peer ipv4:142.251.5.95:443 {created_time:"2023-04-27T13:39:25.291980665+00:00", grpc_status:3, grpc_message:"Request contains an invalid argument."}"
>, <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.INVALID_ARGUMENT
	details = "Request contains an invalid argument."
	debug_error_string = "UNKNOWN:Error received from peer ipv4:142.251.5.95:443 {created_time:"2023-04-27T13:39:25.291980665+00:00", grpc_status:3, grpc_message:"Request contains an invalid argument."}"
>, errors {
  error_code {
    query_error: PROHIBITED_SEGMENT_IN_SELECT_OR_WHERE_CLAUSE
  }
  message: "Cannot select or filter on the following segments: \'segments.date\'(could not support requested resources: \'ASSET\'), since segment is incompatible with the resource in the FROM clause or other selected segmenting resources."
}
request_id: "DklbV4tR664Wb_274bja-A"
, 'DklbV4tR664Wb_274bja-A')
```

### Contribute

- [X] Yes, I want to contribute

### PR DESCRIPTION
**Pull request by bibimorlet**
## What
Closes #25620 
- The 'asset' Resource from Google Ads is not yet handled by the connector.

## How
- add 'asset' Resource in FULL_REFRESH_CUSTOM_TABLES

## Pre-merge Checklist
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Connector version has been incremented
    - [x] Version has been bumped according to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines
    - [x] `Dockerfile` has updated version
- [x] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] Changelog updated in `docs/integrations/<source or destination>/<name>.md` with an entry for the new version. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)

- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [x] Create a non-forked branch based on this PR and test the below items on it
- [x] Build is successful
- [x] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [x] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [x] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
</details>

### CODE REVIEW

**Comment by bibimorlet** [comment]
What should I do in order to make the "Add labels to github PRs based on context" check successful?

File: docs/integrations/sources/google-ads.md [hunk 1]
```diff
@@ -152,10 +152,12 @@ Due to a limitation in the Google Ads API which does not allow getting performan
...
+| `0.2.20` | 2023-03-27 | [25624](https://github.com/airbytehq/airbyte/pull/25624) | Add `asset` Resource to full refresh custom tables (GAQL Queries)                                                                    |
 | `0.2.19` | 2023-05-15 | [26209](https://github.com/airbytehq/airbyte/pull/26209) | Handle Token Refresh errors as `config_error`                                                                                        |
 | `0.2.18` | 2023-05-15 | [25947](https://github.com/airbytehq/airbyte/pull/25947) | Improve GAQL parser error message if multiple resources provided                                                                     |
 | `0.2.17` | 2023-05-11 | [25987](https://github.com/airbytehq/airbyte/pull/25987) | Categorized Config Errors Accurately                                                                                                 |
 | `0.2.16` | 2023-05-10 | [25965](https://github.com/airbytehq/airbyte/pull/25965) | Fix Airbyte date-time data-types                                                                                                     |
+>>>>>>> master
```
Comment (sh4sh) [inline docs/integrations/sources/google-ads.md:160]: My bad

```suggestion
```

**Review by lazebnyi — APPROVED** [review]

**Pull request merged**